### ISSUE TITLE
Fatal error: exception Parse_error.Error(_)

### ISSUE THREAD

**Issue by jhgg** [facebook/flow#559]
Just updated to flow v0.12.0, and am getting this error when I run `flow check`. 

```
git:master ❯❯❯ flow check                                                                                    ✭ ✱ ◼
Fatal error: exception Parse_error.Error(_)
```

Unfortunately I don't know where to go from here, I've tried running `flow check --debug`, and all other debug related flags, but no output aside from that weird parse error.

How would I diagnose this further?

Flow config:

```
[ignore]

[include]
./src

[libs]

[options]
```

**Comment by samwgoldman** [comment]
I'm 99% sure you would get a better error message once #538 is merged.

If I'm right, the issue is an invalid package.json file somewhere in node_modules. Does `find . -name invalidPackageJson` return anything in your project?

**Comment by jeffmo** [comment]
/me waves the magic merge wand on #538

**Comment by mkatrenik** [comment]
@jhgg I resolved it by:
```
[ignore]
.*/node_modules/invalidPackageJson.*
```

@samwgoldman for example it's in webpacks deps

**Comment by jhgg** [comment]
@samwgoldman, yes:
```
❯❯❯ find . -name invalidPackageJson               ⏎ ✭ ✱ ◼
./node_modules/webpack/node_modules/enhanced-resolve/test/fixtures/node_modules/invalidPackageJson
./node_modules/webpack-dev-server/node_modules/webpack-dev-middleware/node_modules/enhanced-resolve/test/fixtures/node_modules/invalidPackageJson
```

@mkatrenik, that didn't resolve it. I also tried:

```
.*/node_modules/.*invalidPackageJson
```

**Comment by samwgoldman** [comment]
I believe those regex will not match the malformed `package.json`, because `invalidPackageJson` appears in the middle of the path. Something like `.*/invalidPackageJson/.*` should do it, though.

**Comment by samwgoldman** [comment]
Tracking the long-term fix in #561.

**Comment by jacowley** [comment]
None of these workarounds fix the problem for me including ```.*/invalidPackageJson/.*```. I'm also finding the invalidPackageJson in webpack's enhanced-resolve dependency.

**Comment by istarkov** [comment]
+1 the same problem

**Comment by samwgoldman** [comment]
It sounds like people are still having issues even using the `[ignore]` workaround, so I'll keep this one open. However, please try again with 0.13, because that release gives a more informative error message. Perhaps there are just more paths that need to be ignored?

**Comment by jhgg** [comment]
I think that a less cryptic error message with steps to remedy the issue, including a list of files that perhaps need to be ignored which I think is happening in 0.13 is enough to solve this issue.

**Comment by mroch** [comment]
been a long time since I've seen an issue like this, so going to close this. but please follow up or create a new issue if this still repros
